docs/en/connector/flink-sql/Kafka.md

```diff
@@ -0,0 +1,115 @@
+# Flink SQL Kafka Connector
+
+## Description
+
+With kafka connector, we can read data from kafka and write data to kafka using Flink SQL. Refer to the [Kafka connector](https://nightlies.apache.org/flink/flink-docs-release-1.13/docs/connectors/table/kafka/) for more details.
+
+
+## Usage
+Let us have a brief example to show how to use the connector from end to end.
+
+### 1. kafka prepare
+If you have the kafka environment, you can directly go to step 3.
+
+We can download kafka distribution from [Apache Kafka](https://kafka.apache.org/downloads.html).
+
+After decompressing the kafka distribution, we can get the following files.
+```bash
+$ ls
+LICENSE   NOTICE    bin       config    libs      licenses  logs      site-docs
+```
+
+Before starting kafka server, we should start the zookeeper server by executing the following command.
+```bash
+$ bin/zookeeper-server-start.sh config/zookeeper.properties
+```
+
+Open another terminal and start kafka server.
+```bash
+$ bin/kafka-server-start.sh config/server.properties
+```
+
+If the shell process not exit, it means kafka server start successfully.
+
```

Comment:
Recommend removing this part, you can add `https://kafka.apache.org/quickstart` a link here, otherwise we need to sync with Kafka site.

Reply:
good catch